.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: true
```

Comment:
hmm i have to say that this library is such easy to use we should invest that hour to make it work (@stemod i think its not more time)